x-pack/test/security_solution_endpoint/apps/endpoint/endpoint_list.ts

```diff
@@ -99,7 +98,9 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       after(async () => {
         await deleteAllDocsFromMetadataCurrentIndex(getService);
         await deleteAllDocsFromMetadataUnitedIndex(getService);
-        await endpointTestResources.unloadEndpointData(indexedData);
+        if (indexedData) {
```

Comment:
Out of curiosity, why is this needed now?

Reply:
Got it, I'm ok with that but would be good try to know why sometimes there is data and why sometimes there isn't just to avoid having hidden bugs because this